src/Mixer/Mixer.cpp

```diff
@@ -0,0 +1,109 @@
+#include "NAS2D/Mixer/Mixer.h"
+
+namespace NAS2D
+{
+
+	Mixer::Mixer(const std::string& name)
+	: mName(name)
+	{
+	}
+
+	Mixer::Mixer()
+	{
+	}
+
+	Mixer::~Mixer()
+	{
+	}
+
+	void Mixer::playSound(Sound& /*sound*/)
+	{
+	}
+
+	void Mixer::stopSound()
+	{
+	}
+
+	void Mixer::pauseSound()
+	{
+	}
+
+	void Mixer::resumeSound()
+	{
+	}
+
+	void Mixer::playMusic(Music& music, int loops /*= Mixer::CONTINUOUS*/)
+	{
+		fadeInMusic(music, loops, 0);
+	}
+
+	void Mixer::stopMusic()
+	{
+	}
+
+	void Mixer::pauseMusic()
+	{
+	}
+
+	void Mixer::resumeMusic()
+	{
+	}
+
+	void Mixer::fadeInMusic(Music& /*music*/, int /*loops*/ /*= Mixer::CONTINUOUS*/, int /*time*/ /*= Mixer::DEFAULT_FADE_TIME*/)
+	{
+	}
+
+	void Mixer::fadeOutMusic(int /*time*/ /*= Mixer::DEFAULT_FADE_TIME*/)
+	{
+	}
+
+	bool Mixer::musicPlaying() const
+	{
+		return false;
+	}

```

Comment:
This method isn't completely empty, but mostly exists to satisfy the requirement to return a value. As the default isn't likely useful to most derived classes, this would also be better off as pure virtual.